docs/user/openstack/install_upi.md

```diff
@@ -336,22 +354,35 @@ values:
 
 <!--- e2e-openstack-upi: INCLUDE START --->
 ```sh
-$ python -c 'import yaml
+$ python -c 'import os
+import yaml
+import re
+re_os_net_id = re.compile(r"{{\s*os_net_id\s*}}")
+os_net_id = os.getenv("OS_NET_ID")
+path = "common.yaml"
+facts = None
+for _dict in yaml.safe_load(open(path))[0]["tasks"]:
+    if "os_network" in _dict.get("set_fact", {}):
+        facts = _dict["set_fact"]
+        break
+os_network = re_os_net_id.sub(os_net_id, facts["os_network"])
```

Comment:
Now what happens if `common.yaml` stop having a task that sets an `os_network` fact? Ideally we would error out in a clean way, with an actionable error message.

Reply:
In case, there is no section in `common.yaml` containing `os_network`, script will fail with traceback on line 368, as facts will be `None` in that case.

I'll add error handling, although, I have a feeling, that we starting to spent too much time on handling errors in this particular case. As an example - changing inventory by removing localhost to something else will break this snippet in the same way.